### PR TITLE
fix: date has any type

### PR DESCRIPTION
**Pull request by bang88**
This will block ant-mobile enable `noImplicitAny`

### CODE REVIEW

File: src/Popup.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { IPopupPickerProps } from 'rmc-picker/lib/PopupPickerTypes';
 
 export interface IPopupDatePickerProps extends IPopupPickerProps {
   datePicker: React.ReactElement<IDatePickerProps>;
-  onChange?: (date?) => void;
+  onChange?: (date?: any) => void;
```
Comment (paranoidjk) [inline src/Popup.tsx]: should there be a Date Object?

Comment (bang88) [reply src/Popup.tsx]: not sure I just need a quick fix so we can merge `refactor-ts`

Comment (paranoidjk) [reply src/Popup.tsx]: pretty sure it should be Date.

**Comment by bang88** [comment]
Ping @warmhug or @paranoidjk

**Comment by paranoidjk** [comment]
`+ m-data-picker@6.0.6`

**Pull request merged**